app/config.py

```diff
@@ -103,13 +103,6 @@ class Decoupled(Config):
     PLANNED_TESTS_YAML_FILE_NAME = "planned-tests.yaml"
```

Comment:
With regards to these files I think we should either be removing the environmentally-conditional ones (i.e., "planned-tests-<dev|preview|staging>.yaml" or changing this to something like PLANNED_TESTS_YAML_FILE_NAME = `f"planned-tests-{os.getenv('ENVIRONMENT')}.yaml"`
It was previously desired to see what planned tests would look like across environments, but this may have been at a time where we didn't have manual approval of production code, thus they may be redundant files now.